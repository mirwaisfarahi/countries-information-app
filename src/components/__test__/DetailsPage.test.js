import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import DetailsPage from '../DetailsPage';
import store from '../../redux/store/configureStore';
import '@testing-library/jest-dom';

it('Check if Item component has changed', () => {
  const country = {
    name: 'Iraq',
    capital: 'Bagdad',
    flag: 'https//flag',
    timezone: 'utc+1',
  };

  const tree = renderer.create(
    <Provider store={store}>
      <BrowserRouter>
        <DetailsPage country={country} />
        ,
      </BrowserRouter>
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
