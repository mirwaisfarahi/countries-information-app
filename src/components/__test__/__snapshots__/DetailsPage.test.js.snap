// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if Item component has changed 1`] = `
Array [
  <div
    className="country-details-container"
  />,
  ",",
]
`;
